PWT/Red/Adventures.yaml

```diff
@@ -0,0 +1,132 @@
+---
+species: Venusaur
+setname: PWT-Adventures Red
+ingamename: Saur
+displayname: Saur
+tags: [in-game, PWT-Adventures Red, trainer+red]
+item: [null]
+ability: [Overgrow]
+moves:
+- [Frenzy Plant]
+- [Vine Whip]
+- [Poison Powder]
+- [Leech Seed]
+gender: [m]
+happiness: 255
+biddable: false
+ball: [Poke]
+nature: Gentle
+ivs: 0
+evs: {hp: 0, atk: 0, def: 0, spA: 0, spD: 0, spe: 0}
```

Comment:
In cases like this (and I'm not going to point it out each time; there's a ton of these), you can just write `evs: 0` for simplicity. That applies whenever all EVs or IVs are set to the same value.

Reply:
Makes sense. I just copied the PWT files from others which already had those in, so I didn’t need to change them at all. Will adjust anyway for future additions.